### PR TITLE
How to create a gRPC patch file

### PR DESCRIPTION
**Pull request by ffoulkes**

### CODE REVIEW

**Review by 5abeel — APPROVED** [review]
LGTM

File: docs/how-to-create-a-grpc-patch-file.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  ```
+
+Insert a command to set the `INSTALL_RPATH` property between the
+`add_executable()` command and its successor:
+
+  ```bash
+  add_executable(grpc_cpp_plugin
+    src/compiler/cpp_plugin.cc
+  )
+  set_target_properties(grpc_python_plugin
```
Comment (5abeel) [inline docs/how-to-create-a-grpc-patch-file.md]: This should be `grpc_cpp_plugin` and not `grpc_python_plugin`

File: docs/how-to-create-a-grpc-patch-file.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    PROPERTIES INSTALL_RPATH @GRPC_INSTALL_RPATH@
+  )
+  target_compile_features(grpc_python_plugin PUBLIC cxx_std_14)
+  ```
+
+Save the modified file.
+
+## Create patch file
+
+Use `diff -u` to generate a patch file.
```
Comment (5abeel) [inline docs/how-to-create-a-grpc-patch-file.md]: This seems to be a problem where the cmake system stalls wondering which file to patch (the CMakeLists.txt or CMakeLists.patched file) and waits for a prompt from user.

I found that generating using a `git diff CMakeLists.txt CMakeLists.patched` results in a compact diff that does not stop for user input 

Instructions here need to be updated. Also needs the example generated file updated

**Review by 5abeel — CHANGES_REQUESTED** [review]

File: docs/how-to-create-a-grpc-patch-file.md (hunk 2 above)
Comment (ffoulkes) [reply docs/how-to-create-a-grpc-patch-file.md]: This is why we have code reviews... 😄

File: docs/how-to-create-a-grpc-patch-file.md (hunk 1 above)
Comment (ffoulkes) [reply docs/how-to-create-a-grpc-patch-file.md]: Oops! 😊

**Review by 5abeel — APPROVED** [review]
LGTM

**Pull request merged**